infrastructure/src/main/java/org/corfudb/infrastructure/LogUnitServer.java

```diff
@@ -81,6 +81,7 @@
     /**
      * The server context of the node.
      */
+    @Getter
```

Comment:
Is this necessary? Looks like no one use it other than itself.

Reply:
Removed